src/client/liquity/trove-bridge-data.ts

```diff
@@ -15,23 +19,42 @@ import { AuxDataConfig, AztecAsset, AztecAssetType, BridgeDataFieldGetters, Soli
 
 export class TroveBridgeData implements BridgeDataFieldGetters {
   public readonly LUSD = EthAddress.fromString("0x5f98805A4E8be255a32880FDeC7F6728C6568bA0");
+  // Price precision
+  public readonly PRECISION = 10n ** 18n;
+
+  public readonly MAX_ACCEPTABLE_SLIPPAGE = 500n; // Denominated in basis points
```

Comment:
Addressed the naming in https://github.com/AztecProtocol/aztec-connect-bridges/pull/286/commits/af5f3d5913f7ee47449f5f163ccc20cc4e6a8b4b 

The rest I think might be a bit of a misunderstanding because how it works is that when there is a batch with slippage setting set to a value in a range <"current LUSD oracle price"; "current LUSD oracle price + MAX_ACCEPTABLE_SLIPPAGE_SETTING"> the setting from the batch gets used. Since the MAX is currently 5 % and IDEAL is 2 %  the users would get batched in the example you provided. Am I missing something?

Reply:
The idea of the example was just to convince myself of why the two slippage % values, 
should not be the same. Say that both slippages are 2%.

Improved example
At time 1, the price is lusd_eth_price = 1/1000
A user wants to repay and uses a slippage of 2%, the max price will be 1.02/1000. 
The batch is initiated.

At time 2, the price is lusd_eth_price = 0.995/1000.
A user wants to repay and uses a slippage of 2%, the max price will be 1.0149/1000
0.995/1000 < 1.02/1000 == true, but 1.02/1000 < 1.0149/1000 == false
The batch is rejected. And a new is initiated.

Instead, say that we are using the 2% and 3%

At time 1, the price is lusd_eth_price = 1/1000
A user wants to repay and uses a slippage of 2%, the max price will be 1.02/1000. 
The batch is initiated.

At time 2, the price is lusd_eth_price = 0.995/1000.
A user wants to repay and uses a slippage of 2%, the max price will be 1.0149/1000
But to consider the batch, we use 3% so batch max price will be 1.02485/1000

0.995/1000 < 1.02/1000 == true, and 1.02/1000 < 1.02485/1000 == true
The batch is accepted. And the user tags along.